### PR TITLE
examples added

### PR DESCRIPTION
**Pull request by Madhan-SWE**
Fixes: https://github.com/ppc64le-cloud/powervs-csi-driver/issues/18

### CODE REVIEW

File: examples/kubernetes/block-volume/specs/pod.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    image: busybox 
+    command: ["/bin/sh", "-c"]
+    args: ["tail -f /dev/null"]
+    volumeDevices:
+    - name: data
+      devicePath: /dev/xvda
+  volumes:
+  - name: data
+    persistentVolumeClaim:
+      claimName: block-claim
```
Comment (mkumatag) [inline examples/kubernetes/block-volume/specs/pod.yaml:18]: add the new line in the end and in all the files wherever required

**Review by mkumatag — CHANGES_REQUESTED** [review]
with all the additional examples, do we still need to examples in the below dir:

```
examples/pvc.yaml
examples/pod.yaml
examples/sc.yaml
```

File: examples/kubernetes/block-volume/specs/pod.yaml (hunk 1 above)
Comment (Madhan-SWE) [reply examples/kubernetes/block-volume/specs/pod.yaml]: fixed

**Comment by Madhan-SWE** [comment]
> with all the additional examples, do we still need to examples in the below dir:
> 
> ```
> examples/pvc.yaml
> examples/pod.yaml
> examples/sc.yaml
> ```

files removed

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**